### PR TITLE
Add support for SVG animations

### PR DESCRIPTION
**Pull request by Trombach**
### Description

Implementation PR: https://github.com/vtbag/element-crossing/pull/5

Adds an animated SVG image to test and demonstrate the new `svg-anim` expression

Todo:

- [x] make tests actually useful (not sure how since it's difficult to compare the animation times as they depend a bit on when playwright actually runs the code)
- [x] amend docs with info about abbreviated form, once decided

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -35,7 +35,7 @@ export default defineConfig({
   projects: [
     {
       name: 'chromium',
-      use: { ...devices['Desktop Chrome'] },
+      use: { ...devices['Desktop Chrome'], channel: 'chrome', },
```
Comment (martrapp) [inline playwright.config.ts:38]: This was necessary to get the video test case running. 
Without it `video.play()` did not work with playwright.

File: tests/01_crossing.spec.ts [hunk 2]
```diff
@@ -54,21 +54,29 @@ test.describe("vtbag", () => {
 
 
   test('preserves animations and media timing', async ({ page }) => {
-    // todo: unable to get the video to play yet
```
Comment (martrapp) [inline tests/01_crossing.spec.ts:57]: was able to get the video case to work and also added a check for CSS animation.

**Review by martrapp — COMMENTED** [review]
Next to the vanilla demo there is also a plain and a over-the-top demo that we should update accordingly by adding the animated SVG.
/crossing/over-the-top/1/
/crossing/plain/1/

**Comment by Trombach** [comment]
> Next to the vanilla demo there is also a plain and a over-the-top demo that we should update accordingly by adding the animated SVG. /crossing/over-the-top/1/ /crossing/plain/1/

Thanks, I missed that. I will add this and also make the required changes once we've agreed on the final implementation for this feature.

**Comment by martrapp** [comment]
Love it! I'm very happy about this contribution! Thank you very much, Lukas!

**Pull request merged**